icons/filezilla/filezilla-line.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M4.783.5c0 1.13-.456 2.232-1.252 3.031A4.324 4.324 0 0 1 .5 4.783v5.653a4.289 4.289 0 0 1 4.283 4.288A4.283 4.283 0 0 1 .5 19.007v5.649a4.29 4.29 0 0 1 4.283 4.283A4.294 4.294 0 0 1 .5 33.23v5.65a4.286 4.286 0 0 1 0 8.574v5.65a4.289 4.289 0 0 1 4.283 4.286A4.289 4.289 0 0 1 .5 61.677v5.65a4.286 4.286 0 0 1 0 8.57v5.65a4.288 4.288 0 0 1 4.283 4.282A4.289 4.289 0 0 1 .5 90.117v5.653a4.285 4.285 0 0 1 0 8.57v5.653a4.283 4.283 0 0 1 4.283 4.283A4.289 4.289 0 0 1 .5 118.564v5.653c1.13 0 2.232.456 3.031 1.252a4.324 4.324 0 0 1 1.252 3.031h5.653a4.289 4.289 0 0 1 4.288-4.283 4.283 4.283 0 0 1 4.283 4.283h5.649a4.289 4.289 0 0 1 4.287-4.283 4.289 4.289 0 0 1 4.287 4.283h5.65a4.285 4.285 0 0 1 8.57 0h5.65a4.289 4.289 0 0 1 4.286-4.283 4.289 4.289 0 0 1 4.287 4.283h5.65a4.289 4.289 0 0 1 4.287-4.283 4.289 4.289 0 0 1 4.287 4.283h5.65a4.285 4.285 0 0 1 8.57 0h5.649a4.289 4.289 0 0 1 4.287-4.283 4.289 4.289 0 0 1 4.287 4.283h5.645a4.289 4.289 0 0 1 4.288-4.283 4.288 4.288 0 0 1 4.283 4.283h5.657c0-1.13.457-2.232 1.252-3.031a4.324 4.324 0 0 1 3.031-1.252v-5.65a4.288 4.288 0 0 1 0-8.574v-5.645a4.287 4.287 0 0 1 .004-8.574V90.12a4.285 4.285 0 0 1 0-8.57V75.9a4.289 4.289 0 0 1-4.283-4.286 4.289 4.289 0 0 1 4.283-4.287V61.68a4.289 4.289 0 0 1-4.283-4.287 4.283 4.283 0 0 1 4.283-4.283v-5.653a4.289 4.289 0 0 1-4.283-4.287 4.289 4.289 0 0 1 4.283-4.288V33.23a4.289 4.289 0 0 1-4.283-4.287 4.288 4.288 0 0 1 4.283-4.283v-5.65a4.289 4.289 0 0 1-4.283-4.286 4.288 4.288 0 0 1 4.283-4.284V4.783a4.324 4.324 0 0 1-3.031-1.252A4.324 4.324 0 0 1 124.217.5h-5.653a4.289 4.289 0 0 1-4.288 4.283A4.283 4.283 0 0 1 109.993.5h-5.653a4.288 4.288 0 0 1-4.283 4.283A4.289 4.289 0 0 1 95.77.5h-5.65a4.285 4.285 0 0 1-4.283 4.283A4.297 4.297 0 0 1 81.547.5H75.9a4.289 4.289 0 0 1-4.287 4.283A4.289 4.289 0 0 1 67.327.5h-5.65a4.289 4.289 0 0 1-4.287 4.283A4.289 4.289 0 0 1 53.103.5h-5.65a4.285 4.285 0 0 1-8.57 0H33.23a4.289 4.289 0 0 1-4.287 4.283A4.288 4.288 0 0 1 24.66.5h-5.65a4.29 4.29 0 0 1-4.283 4.283c-2.358 0-4.29-1.921-4.29-4.283Zm30.762 18.837h55.953L87.932 36.12H49l-3.334 15.593h63.874l-2.74 11.193L72.649 96.12c3.094.476 6.07 1.189 9.28 1.189 7.817 0 16.801-2.382 22.876-4.882l-3.122 14.645c-8.929 4.287-14.405 4.646-19.755 4.646-5.826 0-11.424-1.906-17.14-1.906-3.213 0-7.854.24-10.713 2.142l-6.192-10.948 35.234-34.884H42.568l-7.735 36.309H17.928Zm0 0" style="fill:none;stroke-width:1;stroke-linecap:butt;stroke-linejoin:miter;stroke:#000;stroke-opacity:1;stroke-miterlimit:4" transform="rotate(.115) scale(.99225)"/></svg>
```

Comment:
Same here as with line-wordmark.
Follow [this guide](https://github.com/devicons/devicon/issues/1306#issuecomment-1297625294) from step 5. onwards